lib/src/st2110/st_tx_video_session.c

```diff
@@ -2884,6 +2881,13 @@ static int tv_attach(struct mtl_main_impl* impl, struct st_tx_video_sessions_mgr
     s->ring_count /= 2;
   }
 
+  enum mtl_port port;
+  for (int i = 0; i < num_port; i++) {
```

Comment:
Move pacing way detection here because the pacing way need to be known before allocating TX queue in tv_init_hw().